build.gradle

```diff
@@ -211,22 +216,32 @@ tasks.register('aggregatedJavadocs', Javadoc) { aggregated ->
 //  linters for various specific languages or file formats
 //
 
-// Gradle build scripts
-// disabled until <https://github.com/gradle/gradle/issues/6246> is fixed
-/*
 allprojects {
 	apply plugin: 'nebula.lint'
+
+    // We use no deprecated Gradle APIs ourselves, and we'd like to keep it that way.
```

Comment:
Indentation?

Reply:
Oops, inconsistent use of tabs (elsewhere) versus spaces (here). Fixed. Thanks!